spec/motion-spec/matchers/start_with_spec.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::StartWith" do
+  it 'start_with passes when the subject starts with the given string' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.